dracula-theme.jsonc

```
@@ -6,35 +6,49 @@
   "themeValues": {
     // values applied to the entire app
     "root": {
+      "current__line": "#44475a",
+      "background": "#282a36",
+      "foreground": "#f8f8f2",
+      "comment": "#6272a4",
+      "pink": "#ff79c6",
       "red": "#ff5555",
       "orange": "#ffb86c",
       "yellow": "#f1fa8c",
       "green": "#50fa7b",
-      "purple": "#bd93f9",
-      "pink": "#ff79c6",
-      "cyan": "#8be9fd",
-      "app__bg0": "#282a36",
-      "comment": "#6272a4",
+      "blue": "#bd93f9",
+      "ltblue": "#8be9fd",
+      "app__bg0": "lighten(@background, 5%)",
       "toolbar__bg0": "lighten(saturate(@app__bg0, 3%), 6%)",
       "toolbar__bg1": "lighten(@toolbar__bg0, 4%)",
       "toolbar__bg2": "lighten(@toolbar__bg1, 6%)",
-      "panel__bg0": "lighten(@app__bg0, 5%)",
+      "panel__bg0": "lighten(@background, 5%)",
       "panel__bg1": "lighten(@panel__bg0, 5%)",
       "panel__bg2": "lighten(@panel__bg1, 5%)",
-      "input__bg": "fade(@app__bg0, 20%)",
+      "input__bg": "fade(@background, 10%)",
       "input-bg-warn-color": "fade(@orange, 60%)",
-      "panel-border": "darken(@app__bg0, 4%)",
-      "section-border": "fade(#44475a, 8%)",
-      "subtle-border": "fade(#44475a, 4%)",
-      "modal-overlay-color": "rgb(40, 42, 54)",
+      "panel-border": "darken(@background, 4%)",
+      "section-border": "fade(@foreground, 8%)",
+      "subtle-border": "fade(@foreground, 4%)",
+      "modal-overlay-color": "@background",
+      // graph colors
+      "graph-color-0": "@ltblue",
+      "graph-color-1": "@pink",
+      "graph-color-2": "@blue",
+      "graph-color-3": "@orange",
+      "graph-color-4": "@comment",
+      "graph-color-5": "fade(@ltblue, 8%)",
+      "graph-color-6": "fade(@pink, 8%)",
+      "graph-color-7": "fade(@blue, 8%)",
+      "graph-color-8": "fade(@orange, 8%)",
+      "graph-color-9": "fade(@comment, 8%)",
```

Why change the order of the graph colors? We should keep it consistent unless there is a good reason for the change. I do like the fade for the later colors.
```suggestion
      "graph-color-0": "@purple",
      "graph-color-1": "@cyan",
      "graph-color-2": "@green",
      "graph-color-3": "@orange",
      "graph-color-4": "@yellow",
      "graph-color-5": "fade(@purple, 8%)",
      "graph-color-6": "fade(@cyan, 8%)",
      "graph-color-7": "fade(@green, 8%)",
      "graph-color-8": "fade(@orange, 8%)",
      "graph-color-9": "fade(@yellow, 8%)",
```